src/popup/pages/TransactionDetails.vue

```diff
@@ -365,17 +363,14 @@ export default {
     }
   }
 
-  .copy-hash::v-deep {
-    .copied {
-      align-items: center;
+  .address {
+    @extend %face-sans-12-medium;
 
-      .text {
-        flex: 0;
-        margin-inline: 8px;
-        word-spacing: 100vw;
-        justify-content: center;
-      }
-    }
+    text-align: left;
+    letter-spacing: 0.05em;
+    line-height: 18px;
+    color: variables.$color-light-grey;
+    word-break: break-all;
```

Comment:
same exact code is in `src/popup/components/Overview.vue`. Maybe we could extract it somewhere?

Reply:
OK